packages/tkg-clusterclass-docker/bundle/config/upstream/base.yaml

```diff
@@ -148,10 +148,10 @@ spec:
           jsonPatches:
             - op: add
               path: "/spec/template/spec/kubeadmConfigSpec/initConfiguration/nodeRegistration/kubeletExtraArgs/cgroup-driver"
-              value: cgroupfs
+              value: systemd
```

Comment:
What's the reason for this? Seems like this is already conditionally setting the right value for the cgroup-driver based on the kubernetes version.

Reply:
This mr is just making some simple change to test whether it works for the pipelines :)

I saw you posted #3933 and it's much more well formed. Will close this pr and let's track it in #3933